resources/logging/charts/fluent-bit/templates/configmap.yaml

```diff
@@ -0,0 +1,227 @@
+        Exclude_Path      {{ .Values.config.inputs.tail.excludePath }}
+        {{- end }}
+        Parser            {{ .Values.config.inputs.tail.parser }}
+        Docker_Mode       {{ .Values.config.inputs.tail.dockerMode }}
+        Docker_Mode_Flush {{ .Values.config.inputs.tail.dockerModeFlush }}
+        DB                {{ .Values.config.inputs.tail.db }}
+        Mem_Buf_Limit     {{ .Values.config.inputs.tail.memBufLimit }}
+        Skip_Long_Lines   {{ .Values.config.inputs.tail.skipLongLines }}
+        Refresh_Interval  {{ .Values.config.inputs.tail.refreshInterval }}
+        {{- if .Values.config.inputs.tail.bufferChunkSize }}
+        Buffer_Chunk_Size {{ .Values.config.inputs.tail.bufferChunkSize }}
+        {{- end }}
+        {{- if .Values.config.inputs.tail.bufferMaxSize }}
+        Buffer_Max_Size   {{ .Values.config.inputs.tail.bufferMaxSize }}
+        {{- end }}
+
+{{- if .Values.config.outputs.loki.enabled }}
+    [INPUT]
+        Name              tail
+        Tag               loki.*
+        Path              {{ .Values.config.inputs.tailLoki.path }}
+        {{- if or .Values.config.inputs.tailLoki.exclude.namespaces .Values.config.inputs.tailLoki.excludePath}}
+        Exclude_Path      {{ include "loki.namespace.filter" $ | trimSuffix ", " }}
+        {{- end }}
+        Parser            {{ .Values.config.inputs.tailLoki.parser }}
+        Docker_Mode       {{ .Values.config.inputs.tailLoki.dockerMode }}
+        Docker_Mode_Flush {{ .Values.config.inputs.tailLoki.dockerModeFlush }}
+        DB                {{ .Values.config.inputs.tailLoki.db }}
+        Mem_Buf_Limit     {{ .Values.config.inputs.tailLoki.memBufLimit }}
+        Skip_Long_Lines   {{ .Values.config.inputs.tailLoki.skipLongLines }}
+        Refresh_Interval  {{ .Values.config.inputs.tailLoki.refreshInterval }}
+        {{- if .Values.config.inputs.tailLoki.bufferChunkSize }}
+        Buffer_Chunk_Size {{ .Values.config.inputs.tailLoki.bufferChunkSize }}
+        {{- end }}
+        {{- if .Values.config.inputs.tailLoki.bufferMaxSize }}
+        Buffer_Max_Size   {{ .Values.config.inputs.tailLoki.bufferMaxSize }}
+        {{- end }}
+{{- end }}
+
+{{ if .Values.config.inputs.additional }}
+{{- toString .Values.config.inputs.additional | indent 4 }}
+{{- end }}
+
+    [FILTER]
+        Name                kubernetes
+        Match               {{ .Values.config.filters.kubernetes.match }}
+        Kube_URL            https://kubernetes.default.svc:443
+        Merge_Log           {{ .Values.config.filters.kubernetes.mergeLog }}
+        Keep_Log            {{ .Values.config.filters.kubernetes.keepLog }}
+        {{- if .Values.config.filters.kubernetes.mergeLogKey }}
+        Merge_Log_Key       {{ .Values.config.filters.kubernetes.mergeLogKey }}
+        {{- end }}
+        K8S-Logging.Parser  {{ .Values.config.filters.kubernetes.parser }}
+        K8S-Logging.Exclude {{ .Values.config.filters.kubernetes.exclude }}
```

Comment:
I rendered the template before and after this change and found out that there was a small difference:

```
    [FILTER]                                                         [FILTER]
        ...                                                              ...
        K8S-Logging.Parser  On                                           K8S-Logging.Parser  true
        K8S-Logging.Exclude On                                           K8S-Logging.Exclude true
```
Not sure if it's important though (even though in the docs they use On instead of true for booleans)

Reply:
Fixed.